Doesn't build with latest openrct2.d.ts

When using the latest types from `openrct2.d.ts` you get the following build errors.

```
> openrct2-title-editor@1.2.0 build
> tsc

src/main.ts(23,16): error TS2345: Argument of type '{ name: string; version: string; authors: string[]; type: "intransient"; licence: string; main: () => void; }' is not assignable to parameter of type 'PluginMetadata'.
  Property 'targetApiVersion' is missing in type '{ name: string; version: string; authors: string[]; type: "intransient"; licence: string; main: () => void; }' but required in type 'PluginMetadata'.
src/titleEditorWindow.ts(504,13): error TS2322: Type 'number | undefined' is not assignable to type 'number'.
  Type 'undefined' is not assignable to type 'number'.
```

Also worth noting that the readme in this project doesn't tell you about how to use `openrct2.d.ts`. I had to copy it to the project root.

edit: Turns out this is because the project is currently set up to to get it from [here](https://github.com/OpenRCT2/title-sequence-editor/blob/main/node_modules/%40types/openrct2/index.d.ts), but that doesn't work since `npm install` clears the whole directory.

--- a/src/main.ts
+++ b/src/main.ts
@@ -26,5 +26,6 @@ registerPlugin({
     authors: ['OpenRCT2'],
     type: 'intransient',
     licence: 'MIT',
+    targetApiVersion: 52,
     main: main
 });

--- a/src/titleEditorWindow.ts
+++ b/src/titleEditorWindow.ts
@@ -500,7 +500,7 @@ class TitleEditorWindow {
         }
 
         const seqDropdown = this.window.findWidget<DropdownWidget>('dropdown-sequence');
-        if (seqDropdown) {
+        if (seqDropdown && this.currentSequence !== undefined) {
             seqDropdown.selectedIndex = this.currentSequence;
         }
     }

